packages/terra-toolkit-docs/src/terra-dev-site/tool/webpackConfigTerra/About.a.tool.mdx

```diff
@@ -0,0 +1,189 @@
+import { Badge } from '@cerner/webpack-config-terra/package.json?dev-site-package';
+
+<Badge />
+
+# Webpack Config Terra
+
+The webpack-config-terra package contains the base webpack config required to build terra applications. [Webpack](https://webpack.js.org/) is a module bundler used to compile modules with dependencies and generate static assets. Webpack is a very powerful tool that is highly configurable and Terra components rely on specific polyfills, webpack loaders and plugins to render correctly.
+
+## Terra's Configuration Requirements
+
+Below is the list of polyfills, webpack loaders and plugins Terra components rely on:
+
+### Polyfills
+
+React 16 depends on the collection types ``Map`` and ``Set`` and it depends on ``requestAnimationFrame``, so Terra UI needs a polyfilled environment.
+
+- [core-js](https://github.com/zloirock/core-js) - Provides polyfills necessary for ECMAScript: promises, symbols, collections, iterators, typed arrays, many other features, ECMAScript proposals, some cross-platform WHATWG / W3C features and proposals like URL.\*
+- [regenerator-runtime](https://github.com/facebook/regenerator/tree/master/packages/regenerator-runtime) - Transformer for enabling ECMAScript 6.\*
+
+### JavaScript Loaders
+
+- [babel-loader](https://webpack.js.org/loaders/babel-loader/) - Allows transpiling JavaScript files using [Babel](https://github.com/babel/babel) and webpack.
+- [file-loader](https://webpack.js.org/loaders/file-loader/) - Instructs webpack to emit the required object as file and to return its public URL.
+
+### JavaScript Plugins
+
+- [DefinePlugin](https://webpack.js.org/plugins/define-plugin/) - Plugin to define global compile-time values, including:
+  - `CERNER_BUILD_TIMESTAMP` - The time that webpack was executed in ISO8601 format.
+  - `TERRA_AGGREGATED_LOCALES` - The list of successfully aggregated locales available to the browser.
+  - `TERRA_THEME_CONFIG` - The scoped and default theme from
```

Comment:
Can we describe this object structure in more detail?

Reply:
Yes, I must have gotten distracted here and didn't finish my thought.